### PR TITLE
Compatibility shim for common gate

### PR DESCRIPTION
**Pull request by mpharrigan**
fixes https://github.com/quantumlib/Cirq/pull/2205#issuecomment-539260440

I need this. Feel free to take an add the rest of the moved classes

### CODE REVIEW

File: cirq/ops/common_gates.py [hunk 1]
```diff
@@ -37,6 +38,33 @@
 from cirq.type_workarounds import NotImplementedType
 
 
+def SWAP(*args, **kwargs):
```
Comment (Strilanc) [inline cirq/ops/common_gates.py]: This is only a half-working shim. For example, `type(cirq.ops.common_gates.SWAP)` will return something different.

I recommend just setting `from cirq.ops.swap_gate import SWAP` with a compatibility note. We will figure out how to deprecate it later.

Comment (mpharrigan) [reply cirq/ops/common_gates.py]: done

**Comment by mpharrigan** [comment]
windows doesn't print the failures

**Comment by Strilanc** [comment]
It does. This is the multithreading flake, not a real error.

**Pull request merged**